lib/sms.rb

```diff
@@ -0,0 +1,42 @@
+require 'twilio-ruby'
+
+class SMS
+
+  TIME_FORMAT = "%H:%M"
+
+  CONFIG =
+    { account_sid: "AC1a757e2dd73b62efcbb52def2d72432d",
+      auth_token: "e83f862f205340368bfb1e7a17c3e4c1",
+      from: "+441443606449",
+      to: "+447783980181",
+      body: "Thank you! Your order was placed and will be delivered before %s"
+    }
+
+  def initialize(config = CONFIG, client: nil)
```

Comment:
mixing different types of arguments here is confusing. In this case I would use either keyword arguments for both or not use keyword arguments at all.